package.json

```diff
@@ -7,6 +7,7 @@
   "devDependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
     "bootstrap": "^4.6.1",
+    "copy-props": ">=2.0.5",
```

Comment:
I don't think we should be adding this as an explicit dependency - should just be a case of updating the lock file, or updating the dependency that uses this.